gulpfile.js

```diff
@@ -148,6 +148,11 @@ gulp.task('images', function imagesTask() {
     .pipe(gulp.dest(DEST_PATH + 'images'));
 });
 
+gulp.task('scripts', function imagesTask() {
```

Comment:
We already have [a `scripts` task that builds our minified app.js bundle](https://github.com/kyoshino/testpilot/blob/3f98f4a20ece73fbdc8600a2f82d59412381aea7/gulpfile.js#L83). We should rename this, and change the `imagesTask` function name as a nitpick.